### ISSUE TITLE
Alt Text missing when inserting asset via image button

### ISSUE THREAD

**Issue by sgtpenguin** [craftcms/ckeditor#563]
### Description
When I include an image in CK using the image button, it should populate alt text if it already exists on the asset in the `alt` field.

**Comment by brandonkelly** [comment]
Thanks for pointing that out! CKEditor 5.7.0 is out now with a solution (#585).
